common_dwp_api/src/main/java/uk/gov/dhsc/htbhf/dwp/model/QualifyingReason.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.dhsc.htbhf.dwp.model;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+import java.util.Locale;
+
+public enum QualifyingReason {
+    UNDER_18,
+    NOT_SET;
```

Comment:
As discussed with @mikeSmith08 @AmarkumarJagtap and Karen, QualifyingReason should have _all_ the values form QualifyingBenefits, plus under_18. It will be the job of the eligibility service to translate from the QualifyingBenefit that is returned BY DWP/HMRC into a QualifyingReason sent back to the claimant service. 
The EligibilityOverride applied by the claimant servcie should use QualifyingReason, not QualifyingBenefits.